staging/src/k8s.io/kubelet/config/v1beta1/types.go

```diff
@@ -966,6 +971,14 @@ type MemorySwapConfiguration struct {
 	SwapBehavior string `json:"swapBehavior,omitempty"`
 }
 
+type CrashLoopBackOffConfig struct {
+	// maximumBackOffPeriod is the maximum duration the backoff delay can accrue
+	// to for container restarts, minimum 1 second, maximum 300 seconds.
```

Comment:
why do we need a maximum 300 seconds? Why not unbounded?

Reply:
I think this is a valid idea in principle (though in practice -- I don't know if any people are really looking for LONGER backoffs), but at least for this first alpha round of implementation, it minimizes the real change space (similar reasoning for maintaining the recovery expiration 2x*original maxDuration aka 2x*300=600s=10 minutes). IMHO since it's easier to relax later I prefer leaving this here at this time.